test/test_browser.py

```diff
@@ -5403,6 +5403,14 @@ def test_pthread_unhandledrejection(self):
   def test_full_js_library_strict(self):
     self.btest_exit(test_file('hello_world.c'), args=['-sINCLUDE_FULL_LIBRARY', '-sSTRICT_JS'])
 
+  # Tests the AudioWorklet demo
+  @also_with_minimal_runtime
+  def test_audio_worklet(self):
+    self.btest_exit(test_file('webaudio/audioworklet.c'), args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
+    self.btest_exit(test_file('webaudio/audioworklet.c'), args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '--closure', '1', '-Oz'])
+    self.btest_exit(test_file('webaudio/audioworklet.c'), args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sUSE_PTHREADS'])
+    self.btest_exit(test_file('webaudio/audioworklet.c'), args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sUSE_PTHREADS', '--closure', '1', '-Oz'])
+
```

Comment:
It's better to use `@parameterized` for stuff like this. That way there is a separate function for each test mode, and a failure tells us which failed by the name. It's also easy to run that specific failing one instead of all of them at once.

Reply:
Check